mobile/lib/common/settings/emergency_kit_screen.dart

```diff
@@ -120,6 +120,42 @@ class _EmergencyKitScreenState extends State<EmergencyKitScreen> {
                           Icon(FontAwesomeIcons.broom),
                           SizedBox(width: 10),
                           Text("Cleanup filling orders", style: TextStyle(fontSize: 16))
+                        ])),
+                    const SizedBox(
```

Comment:
🙃 nit: The severity of this feature does not match with the the other emergency kits. I get why you've added it, as its convenient for testing, but I don't think a user will ever have to click that button. So maybe remove it or, only make it visible when on regtest? :)

Reply:
I made it only visible for regtest